pkg/healthcheck/fixers.go

```diff
@@ -84,7 +84,14 @@ func CreateDirectory(path string) Fixer {
 // NotImplemented is a placeholder Fixer which always returns successfully.
 func NotImplemented() Fixer {
 	return func() (string, error) {
-		return "fix is not implemented", nil
+		return "fix is not implemented.", nil
+	}
+}
+
+// NotFixable is a placeholder Fixer which always returns unsuccessfully.
+func NotFixable() Fixer {
```

Comment:
I'd rename this to: `RequiresManualFixing()`, and change error message accordingly for the sake of UX. Always better to provide a "next suggested action" to the user.